.github/workflows/release.yml

```diff
@@ -27,11 +27,11 @@ env:
   SOURCE_DOCKER_REPO: 'us-docker.pkg.dev/github-metrics-aggreg-i-64d426/ci-images'
   TARGET_DOCKER_REPO: 'us-docker.pkg.dev/abcxyz-artifacts/docker-images'
   IMAGE_NAME: 'github-metrics-aggregator'
-  SOURCE_TAG: '${{ github.event.head_sha }}'
+  SOURCE_TAG: '${{ github.event.workflow_run.head_sha }}'
```

Comment:
Why do we want workflow_run instead of push?

Reply:
Yes, hopefully my comment in #318 explains why I went with triggering the release of the cir workflow instead of a push to main. If not, I'm happy to sync and discuss further.